chain/chunks/src/lib.rs

```diff
@@ -467,16 +468,26 @@ impl ShardsManager {
                     added: Instant::now(),
                 },
             );
-            self.request_partial_encoded_chunk(
+            let request_result = self.request_partial_encoded_chunk(
                 height,
                 &parent_hash,
                 shard_id,
                 &chunk_hash,
                 false,
                 false,
-            )?;
+            );
+            if let Err(err) = request_result {
+                error!(target: "chunks", "Error during requesting partial encoded chunk: {}", err);
+                // All errors are logged, but only the first one is returned.
+                if let None = maybe_error {
+                    maybe_error = Some(err);
+                }
+            }
+        }
+        match maybe_error {
```

Comment:
It seems that if we rename this as something like `result` and initialize it as `Ok(())`, we don't have to do the conversion.

Reply:
Yes, good idea!